portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,81 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>ROB MARCUS</title>
     <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
   </head>
   <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+    <h1>
+      <a href="index.html" style="text-decoration: none; color: black">
+        ROB MARCUS
+      </a>
+    </h1>
+    
+    <div id="content">    
+        <div id="container">
+            <div class = "containerBox"> 
```

Comment:
for class/id names, choose either camelCase style or dash-name style, but be consistent throughout your codebase.